### PR TITLE
Settings for successful building the Lite UC Docker image

### PR DESCRIPTION
**Pull request by PetarCrepuljaESL**
Modifications for creating smaller Docker image file, part concerning Usage Control.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
+FROM openjdk:11.0.12-jre
+
 # Add Maintainer Info
 LABEL maintainer="gabriele.deluca@eng.it"
 
-#Install whois service
-RUN yum install -y whois
+# Install whois service
+# RUN yum install -y whois
+# RUN set -eux; 	apt-get update; 	apt-get install -y --no-install-recommends
```
Comment (IgorBalog-Eng) [inline Dockerfile:12]: Is this commented line correct if uncommented?
Please add comment what we do with line below (installing whois)

**Review by IgorBalog-Eng — COMMENTED** [review]
Please address comments in PR

File: Dockerfile (hunk 1 above)
Comment (PetarCrepuljaESL) [reply Dockerfile:12]: Line 12 was used in test phase. It is not necessary here.

**Review by IgorBalog-Eng — APPROVED** [review]
Approved PR :)

**Pull request merged**